src/components/presentation/Presentation.tsx

```diff
@@ -36,6 +35,10 @@ export const PresentationComponent = ({ header, type, children, renderNavBar = t
   const userParty = useProfile()?.party;
   const { expandedWidth } = useUiConfigContext();
 
+  const { showProgress } = usePageSettings();
+  const isProgressEnabled = type !== ProcessTaskType.Archived && showProgress;
+  const showProgressBar = isProgressEnabled && showProgress;
```

Comment:
I faintly remember there being a reason for moving these out to their own component (quite possibly re-renders caused by the page settings update causing weird bugs). If there's no good reason for moving it inline into the very centrally important `Presentation` component, I would prefer to keep the separation of concerns that was.